prismic-model/js/articles.js

```diff
@@ -36,6 +36,9 @@ const Article = {
   Metadata: {
     metadataDescription: structuredText('Metadata description', 'single'),
   },
+  'Content relationships': {
```

Comment:
Could we not just add this to where we add series?

Reply:
was thinking it would make more sense to move the series into this tab, rather than have adding a season under the Series tab - wanted to talk to content editors first though

Reply:
Yea, I'd try make as few changes as few changes as possible, as this is super quick turn around so we don't have much time to work through if something is a good idea or not.

Basically trying to reuse rather than create if we can.

Reply:
Do you want to keep this and make sure we get all the relevant bits in here, or just leave it in the main piece for now?

Reply:
I'd prefer to keep it

Reply:
Okie doke - do you know what other fields you want to get in here? Has it been run past the content team?

Reply:
My understanding is that only events and stories have these types of relationship currently (event-series and articles-series). For events the series is added under the main event tab, for articles it's added under it's own series tab. I think these should both probably moved to a Content relationships tab, alongside seasons, i.e. this is where you go to add this content to something. I'll sound out Danny tomorrow at stand up.

Reply:
Perfect. I only ask as we have, as you have pointed out, a few ways of doing this, and if we are going to try consolidate them, let's make sure we do that rather than proliferate them.